terraform/environments/data-platform/lambda.tf

```diff
@@ -289,8 +289,34 @@ module "data_product_create_schema_lambda" {
       action        = "lambda:InvokeFunction"
       function_name = "data_product_create_metadata_${local.environment}"
       principal     = "apigateway.amazonaws.com"
-      source_arn    = "arn:aws:execute-api:${local.region}:${local.account_id}:${aws_api_gateway_rest_api.data_platform.id}/*/${aws_api_gateway_method.create_schema_for_data_product_table_name.http_method}${aws_api_gateway_resource.create_schema_for_data_product_table_name.path}"
+      source_arn    = "arn:aws:execute-api:${local.region}:${local.account_id}:${aws_api_gateway_rest_api.data_platform.id}/*/${aws_api_gateway_method.create_schema_for_data_product_table_name.http_method}${aws_api_gateway_resource.schema_for_data_product_table_name.path}"
     }
   }
 
 }
+
+module "get_schema_lambda" {
+  source                         = "github.com/ministryofjustice/modernisation-platform-terraform-lambda-function?ref=a4392c1" # ref for V2.1
+  application_name               = "get_schema"
+  tags                           = local.tags
+  description                    = "Fetch the schema for a table from S3"
+  role_name                      = "get_schema_role_${local.environment}"
+  policy_json                    = data.aws_iam_policy_document.iam_policy_document_for_get_schema_lambda.json
```

Comment:
need to add in `policy_json_attached = true` following update to the mod plat lambda module